eatngo-customer-api/src/main/resources/application-metrics.yml

```
@@ -0,0 +1,22 @@
+        "[http.server.requests]": true
+        "[circuit_breaker_calls_duration]": true
+      percentiles:
+        "[http.server.requests]": 0.5, 0.95, 0.99
+        "[circuit_breaker_calls_duration]": 0.5, 0.95, 0.99
+      slo:
+        "[http.server.requests]": 100ms, 500ms, 1s, 2s, 5s
+        "[circuit_breaker_calls_duration]": 100ms, 500ms, 1s, 2s, 5s
+  prometheus:
+    metrics:
+      export:
+        step: 15s
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)